### ISSUE TITLE
Fan control on linux ?

### ISSUE THREAD

**Issue by kinooyume** [cleemesser/aero15X-laptop-linux-cuda-config#5]
Hi there !
I also own a aero 15x (v7) and I'm running on arch linux. Did you manage to have fan control ? I tried some solution from [Arch Wiki](https://wiki.archlinux.org/index.php/Fan_speed_control), but nothing works. Event when I try `sensors` cmdline, I don't get any outputs.
Do you have any clue/solution about it ?
Thank's !

**Comment by kiuKisas** [comment]
Unfortunetly, it doesn't work for me. I hust try to set the fan speed at 80% on Windows/smartManager, reboot to arch linux and it doesn't seems to do anything.. But works well back on windows.
Do you have any differencies from the command `sensors` ? Mine is:
```
coretemp-isa-0000
Adapter: ISA adapter
Package id 0:  +53.0°C  (high = +100.0°C, crit = +100.0°C)
Core 0:        +52.0°C  (high = +100.0°C, crit = +100.0°C)
Core 1:        +51.0°C  (high = +100.0°C, crit = +100.0°C)
Core 2:        +51.0°C  (high = +100.0°C, crit = +100.0°C)
Core 3:        +51.0°C  (high = +100.0°C, crit = +100.0°C)

acpitz-virtual-0
Adapter: Virtual device
temp1:        +27.8°C  (crit = +119.0°C)
temp2:        +29.8°C  (crit = +119.0°C)

iwlwifi-virtual-0
Adapter: Virtual device
temp1:        +37.0°C  

pch_skylake-virtual-0
Adapter: Virtual device
temp1:        +50.0°C  


```

**Comment by Soulthym** [comment]
Unfortunately I cannot try it for now, as I recently needed to do a clean install of ubuntu and it doesn't boot for now for some reason, I'll try to fix it soon and tell you my results

**Comment by Soulthym** [comment]
Sure will, not sure I'll need fan control tho since the windows trick seemed to work. I'll keep you updated on the matter

**Comment by herin049** [comment]
@Soulthym 
> Sure will, not sure I'll need fan control tho since the windows trick seemed to work. I'll keep you updated on the matter
Have you found a solution to the issue yet? I am currently have a similar issue where I cannot figure out how to control the fan speeds on Ubuntu with my Aero 15X. If you could let me know that would be awesome!

**Comment by Soulthym** [comment]
@herin049 I haven't found a solution yet sadly. However the default profile Arch gives seems to be good enough for my workload. Also there probably a possibly to change the value of the variable controlling it in the kernel. I just can't test it since I don't and won't have the machine with me for a couple months.

**Comment by kiuKisas** [comment]
I did succeed with [NBFC](https://wiki.archlinux.org/index.php/Fan_speed_control#NBFC) on arch linux, but sinces the "windows trick" finally work, I did used it since.

**Comment by kiuKisas** [comment]
> @kiuKisas did you managed to control the fan speed in your Arch, I'm having this problem as well.
> I use as kernel parameter `acpi_osi=! acpi_osi="Windows 2009"` for dealing with the Graphic mode.

Nope, since the windows trick finally work on my machine, I didn't try anything else

**Comment by rmahnovetsky** [comment]
+1 to the windows trick. I was looking for ages on how to do this but for a linux solution.. BTW I have the gigabyte aero v9 model. The fans are much quieter now for me :)

**Comment by blmhemu** [comment]
@kiuKisas On a completely unrelated note, by graphic mode, do you mean you were able to use bbswitch and optimus-manger ? Also if not, does anyone know if any acpi commands make it work ?
